### PR TITLE
add the invalid train card to the timetable and delete the old one

### PR DESCRIPTION
**Pull request by theocrsb**
Closes part of [this ticket](https://github.com/osrd-project/osrd-confidential/issues/666).

### CODE REVIEW

**Review by Akctarus — COMMENTED** [review]
Tested, LGTM, just to be pixel perfect, the grey border-bottom that we have in .scenario-timetable-toolbar should be a border-top in .filter, but it's not directly linked to this PR

**Comment by theocrsb** [comment]
> Tested, LGTM, just to be pixel perfect, the grey border-bottom that we have in .scenario-timetable-toolbar should be a border-top in .filter, but it's not directly linked to this PR

Done

File: front/public/locales/en/operationalStudies/scenario.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
       "simulation_failed": "Simulation failed",
       "rolling_stock_not_found": "RS not found"
     },
-    "invalidTrains": "The timetable contains invalid trains.",
+    "invalidTrains": "Some trains are invalid.",
```
Comment (SharglutDev) [inline front/public/locales/en/operationalStudies/scenario.json]: No point in mockup (same for fr)

```suggestion
    "invalidTrains": "Some trains are invalid",
```

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss [hunk 2]
```diff
@@ -598,21 +599,23 @@
...
-          gap: 1rem;
-          justify-content: center;
-          padding: 0.5rem;
-          font-size: 0.9rem;
-          font-weight: 500;
-          border-radius: 0.25rem;
-          border: 0.125rem solid var(--red);
-          margin-block: 1rem;
-          color: var(--red);
+      .invalid-trains {
```
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_scenario.scss]: In the UI, the component is before the trains list so I would reproduce that in the css :)

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss [hunk 3]
```diff
@@ -598,21 +599,23 @@
...
+        display: flex;
+        background: linear-gradient(to bottom, white, var(--warning5) 1px);
```
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Is it to reproduce the white horizontal line below the border-top ? Maybe you could rather use a inset box-shadow for that

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss [hunk 4]
```diff
@@ -598,21 +599,23 @@
...
+        height: 3rem;
+        font-size: 1rem;
+        font-weight: 600;
+        line-height: 24px;
```
Comment (SharglutDev) [inline front/src/styles/scss/applications/operationalStudies/_scenario.scss]:
```suggestion
        line-height: 1.5rem;
```

File: front/public/locales/en/operationalStudies/scenario.json (hunk 1 above)
Comment (theocrsb) [reply front/public/locales/en/operationalStudies/scenario.json]: Done

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss (hunk 2 above)
Comment (theocrsb) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss (hunk 3 above)
Comment (theocrsb) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Done

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss (hunk 4 above)
Comment (theocrsb) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Done

**Review by Akctarus — APPROVED** [review]

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested

**Pull request merged**